apps/api/trade-simulator-docker/startup.sh

```diff
@@ -16,9 +16,9 @@ done
 
 # Check if we need to run migrations
 if ! check_db_initialized; then
-    echo "Database not initialized, running migrations..."
-    cd /workdir/apps/api && pnpm db:migrate
-    echo "Migrations completed"
+    echo "Database not initialized, running baseline and migrations..."
+    cd /workdir/apps/api && pnpm db:prepare-production
```

Comment:
and same thing here...i think the container should maintain whatever flow a "normal" production setup (non-legacy, manual migration) will use, so it should stay as `db:migrate`. 

or, we could set some env var like `RUN_BACKFILL=true`, and if that's set, it'll execute `pnpm:prepare-production ` ; else, `pnpm db:migrate`